### ISSUE TITLE
git must be installed before using this plugin

### ISSUE THREAD

**Issue by mikeweinberg** [rdp/specific_install#26]
I started on a new host that did not have git installed- I kept getting the non-specific errors below but installing git fixed the problem. I'm hoping filing this issue saves someone some time wondering why this gem doesn't work for them. 

```
[username@host ~]$ gem specific_install -l https://github.com/username/repository.git -b master
git installing from https://github.com/username/repository.git
attempting ...
attempting rake gemspec...
rake aborted!
No Rakefile found (looking for: rakefile, Rakefile, rakefile.rb, Rakefile.rb)

(See full trace by running task with --trace)
attempting rake gem...
rake aborted!
No Rakefile found (looking for: rakefile, Rakefile, rakefile.rb, Rakefile.rb)

(See full trace by running task with --trace)
attempting rake build...
rake aborted!
No Rakefile found (looking for: rakefile, Rakefile, rakefile.rb, Rakefile.rb)

(See full trace by running task with --trace)
attempting rake package...
rake aborted!
No Rakefile found (looking for: rakefile, Rakefile, rakefile.rb, Rakefile.rb)

(See full trace by running task with --trace)
```

**Comment by zph** [comment]
@mikeweinberg Thanks for the report on this (and sorry for the delay in responding).

We should error out earlier when git's not installed for any operations that require it!

**Comment by zph** [comment]
Pushed a fix for this and released `v0.2.13`.

Thanks for the error report :+1: .

**Comment by rdp** [comment]
Thanks for your help all.
